### PR TITLE
JDownloader2 LXC Script

### PR DESCRIPTION
**Pull request by asylumexp**
## Description

Adds new script to run JDownloader2 and make it accessible via novnc

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] New Script (Develop a new script or set of scripts that are fully functional and thoroughly tested)
- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.
- [x] This change requires a documentation update

### CODE REVIEW

File: ct/jdownloader2.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="JDownloader2"
+var_disk="10"
+var_cpu="1"
+var_ram="1028"
+var_os="debian"
+var_version="11"
```
Comment (tteck) [inline ct/jdownloader2.sh]: Why version 11?

Comment (asylumexp) [reply ct/jdownloader2.sh]: jdk 11 isn't available on bookworm

edit: ill go try 17 and see if it works with it, i originally had some errors with it but found out it wasnt related to the java version but forgot about it so ill go back and test it quickly

**Comment by asylumexp** [comment]
changed to debian 12

**Comment by tteck** [comment]
After careful review, I have decided not to merge it at this time. The user can utilize Docker if the app is needed.

**Pull request closed without merging**